azext_iot/product/params.py

```diff
@@ -0,0 +1,149 @@
+            arg_group='IoT Device Certification',
+            arg_type=get_enum_type(BadgeType)
+        )
+    with self.argument_context('iot product test') as c:
+        c.argument(
+            'working_folder',
+            options_list=['--working-folder', '--wf'],
+            help='The folder to create in current path',
+            arg_group='IoT Device Certification'
+        )
+        c.argument(
+            'auth_type',
+            options_list=['--authentication-type', '--at'],
+            help='How the device will authenticate to testing service Device Provisioning Service',
+            arg_group='IoT Device Certification',
+            arg_type=get_enum_type(AuthType)
+        )
+        c.argument(
+            'provisioning',
+            options_list=['--provisioning'],
+            help='Determines whether the service generates provisioning configuration. '
+            + 'Only applies to SymmetricKey and ConnectionString provisioning types',
+            arg_group='IoT Device Certification'
+        )
+        c.argument(
+            'configuration_file',
+            options_list=['--configuration-file', '--cf'],
+            help='Path to device test JSON file',
+            arg_group='IoT Device Certification'
+        )
+    with self.argument_context('iot product test search') as c:
+        c.argument(
+            'product_id',
+            options_list=['--product-id', '--pi'],
+            help='The submitted product id',
+            arg_group='IoT Device Certification'
+        )
+        c.argument(
+            'registration_id',
+            options_list=['--registration-id', '--ri'],
+            help='The regstration Id for Device Provisioning Service',
+            arg_group='IoT Device Certification'
+        )
+        c.argument(
+            'certificate_name',
+            options_list=['--certificate-name', '--cn'],
+            help='The x509 Certificate Common Name (cn) used for Device Provisioning Service authentication',
+            arg_group='IoT Device Certification'
+        )
+    with self.argument_context('iot product test case') as c:
+        c.argument(
+            'configuration_file',
+            options_list=['--configuration-file', '--cf'],
+            help='The file path for test case configuration JSON',
+            arg_group='IoT Device Certification'
+        )
+    with self.argument_context('iot product test task') as c:
+        c.argument(
+            'task_id',
+            options_list=['--task-id'],
+            help='The generated Id of the testing task',
+            arg_group='IoT Device Certification'
+        )
+        c.argument(
+            'running',
+            options_list=['--running'],
+            help='Get the running tasks of a device test',
+            arg_group='IoT Device Certification',
+            arg_type=get_three_state_flag()
+        )
+        c.argument(
+            'type',
+            options_list=['--type', '-t'],
```

Comment:
not saying its the case here, but you don't _have_ to have a short alias if the full arg isn't that long to type in

Reply:
in case we want to use `-t` for one of test-id/task-id